src/main/java/bio/terra/service/snapshotbuilder/utils/CriteriaQueryBuilder.java

```diff
@@ -101,31 +104,25 @@ FilterVariable generateFilter(SnapshotBuilderDomainCriteria domainCriteria) {
     TableVariable ancestorVariable =
```

Comment:
It could be nice to add some more java doc to this code!

Reply:
I completely agree!